drivers/Makefile

```diff
@@ -9,6 +9,7 @@ INCLUDES = -I../include/ \
 	 -I./dac/ad917x/ad917x_api/ \
 	 -I../projects/ad9361/src/ \
 	 -I./adc/ad9208/ad9208_api \
+	 -I./adc/ad9083/ad9083_api \
```

Comment:
add also:
`-I./projects/ad9083/src/uc \`

if you are using files from the project inside the driver. 

Although this should be avoided as much as possible.